.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "libs/imagesloaded"]
```

Comment:
I'll get a frontend build going with this and see what happens. I did verify that the file this generates is equivalent to the one we have copied into the frontend (ignoring changes to the version number in comments).